### ISSUE TITLE
Feature request: Reverse listview

### ISSUE THREAD

**Issue by MoMannn** [clockbyte/admobadapter#1]
I have tryed your wrapper and it works perfectly but my app is using listview in a different way. I always add new items to the position 0. So user only scrolls if he wants to see the "history" but new items are always added on top. Because of this adds should display on different positions that they currently are. 

I took a sneak peak into your code and started to change it for my preferences as so:
private boolean isAdPosition(int position) {
        //return (position + 1) % (getNoOfDataBetweenAds() + 1) == 0;
        return (getCount() - position) % getNoOfDataBetweenAds()  == 0;
    }

But soon I found out that a lot of functions should be changed for this to work. This is just a request because you know your code and would probably make this change much faster. Otherwise I will modify it and make a new branch for it.

**Comment by clockbyte** [comment]
@MoMannn Hi! Thats' very strange IMHO. I have the very same behavior in my app, all new messages are always put in the top of the history listview, and the native ads are always moved to the expected position automatically! 
The code snippet from my app:
**somewhere on start of an activity (MessageAdapterWrapper defined as a field of an activity, MessageAdapter is an adapter for a message history, lvMessages is a listview, admob_unit_id is a string unit_id):**
       ``` 
MessageAdapterWrapper = new AdmobAdapterWrapper(getActivity());
 //injecting your history adapter to the AdmobAdapterWrapper
        MessageAdapterWrapper.setAdapter(MessageAdapter);
 //show only 1 ad block at the 5th's position
        MessageAdapterWrapper.setNoOfDataBetweenAds(5);
   MessageAdapterWrapper.setLimitOfAds(1);
MessageAdapterWrapper.setAdmobReleaseUnitId(admob_unit_id);
        lvMessages.setAdapter(MessageAdapterWrapper);```

** then you can add you adapter with new messages everywhere like this in your MessageAdapter.java**

``` 
public class MessagesAdapter extends BaseAdapter{ ...
public void addNewMessages(ArrayList<FoundMessage> sortedMessages) {
        if (sortedMessages== null || sortedMessages.size() == 0) {
            return;
        }
//Items- is an ArrayList<FoundMessage> filled up with your messages which will be published to the listview
//adding fresh items to the top of Items
        Items.addAll(0, sortedMessages);
//don't forget to raise this after each change of collection to signal to the AdmobAdapterWrapper to refresh ads position!!!
        notifyDataSetChanged();
    }...
} 
```

Please clarify in which position do you want to show your native ads in listview. I mean maybe there's no need to change a code for that...But if you'll find that you need  to make a branch, please feel free to do it!

**Comment by MoMannn** [comment]
Okey to clarify my request. I want to show native in the 5th position i want the ads to come like this:
(simulation)

position in array | data[id]/ad
0 | data0
-----
0 | data1
1 | data0
----
0 | data2
1 | data1
2 | data0
----
0 | data3
1 | data2
2 | data1
3 | data0
----
0 | data4
1 | data3
2 | data2
3 | data1
4 | data0
----
0 | ad0
1 | data4
2 | data3
3 | data2
4 | data1
5 | data0
----
0 | data5
1 | ad0
2 | data4
3 | data3
4 | data2
5 | data1
6 | data0
----

I hope this clarifies things. The point is that user does not scroll through the listview but he stays at the top and data is refreshed every 10seconds with new add coming. So the add position should change. 
I think the function for isAdd should look something like this in this case:
private boolean isAdPosition(int position) {
return (getCount() - position) % getNoOfDataBetweenAds() == 0;
}

But then I saw that you calculate none ad position and restrict that first x positions don't have an add and soo on that is when I decided to first ask for a solution. 

Thank you for your response and time.
